test/suites/integration/extended_resources_test.go

```diff
@@ -158,6 +163,54 @@ var _ = Describe("Extended Resources", func() {
 		env.ExpectCreatedNodeCount("==", 1)
 		env.EventuallyExpectCreatedNodesInitialized()
 	})
+	It("should provision nodes for a deployment that requests amd.com/gpu", func() {
+		ExpectAMDDevicePluginCreated()
+		DeferCleanup(func() {
+			ExpectAMDDevicePluginDeleted()
+		})
+
+		content, err := os.ReadFile("testdata/amd_driver_input.golden")
```

Comment:
I'm not sure I understand golden in this context. Typically .golden is referred to as something that's checked, not something that's used.

Reply:
Adjusted the file name